source/stylesheets/refills/_cards.scss

```diff
@@ -1,45 +1,55 @@
 .cards {
+  $card-color: $base-font-color-on-box;
+  $card-border: $base-border;
+  $card-background: $base-box-background;
+  $card-header-background: $card-background;
+  $card-hover-background: lighten($card-background, 5%);
+  $card-image-background: #dbd199;
+  $card-image-hover-background: lighten($card-image-background, 5%);
+  $card-margin: $small-spacing;
+
   @include display(flex);
   @include flex-wrap(wrap);
   @include justify-content(space-between);
-}
 
-.card {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-background-color: white !default;
-  $base-spacing: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  $dark-gray: #333 !default;
-  $base-font-color: $dark-gray !default;
-  //////////////////////////////////////////////////////////////////////////////////
+  .card {
+    @include flex-basis(15em);
+    @include flex-grow(1);
+    @include transition (all 0.2s ease-in-out);
+    background-color: $card-background;
+    border: $card-border;
+    border-radius: $base-border-radius;
+    box-shadow: $base-box-shadow;
+    color: $card-color;
+    cursor: pointer;
+    margin: 0 $card-margin $base-spacing $card-margin;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 $card-margin $base-spacing` instead of `0 $card-margin $base-spacing $card-margin`